packages/test/src/test-workflows.ts

```diff
@@ -2528,3 +2534,83 @@ test('Signals/Updates/Activities/Timers - Trace promises completion order - 1.11
     );
   }
 });
+
+test('Default query handler fail activations with reserved names - workflowWithDefaultHandlers', async (t) => {
```

Comment:
Why did you fallback to unit tests for default handlers?

Reply:
this was a while ago, i'm not sure why, i think because it seemed easier at the time to trigger these failures by sending invalid wft?

in any case, i've moved all the testing to `test-integration-workflows` under a few integration tests.